vite.config.js

```diff
@@ -9,6 +9,10 @@ import vuetify from 'vite-plugin-vuetify';
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
+const PROXY_HOST = process.env.CUTTLE_DOCKERIZED === true
+  ? 'http://localhost:1337'
+  : 'http://cuttle-server:1337';
```

Comment:
What is going on with this? My understanding is that it's using vite to proxy requests from localhost:8080 to either http://localhost:1337 or to http://cuttle-server:1337 if we're using docker?

I'm fuzzy on what this does and when it's used. I tried going to http://cuttle-server:1337 while the container was up and it didn't respond, so I'm not sure how the container name actually relates to the url if at all

Reply:
This is a really interesting nuance, but you have to consider vite is running in the context of docker, and inside of docker the way the services talk to each other is via reference-- so `http://cuttle-server:1337/` actually works when the vue docker image is the one running it. This allows vite to proxy requests from vue (via docker) to sails (via docker) using docker networking.